### DISCUSSION TITLE
Cant get vsCode to open codespace

### DISCUSSION THREAD

**Discussion by gottem87** [community/community#33380] [Codespaces]
On vs code, i try to open a codespace, but it always says, "this repository isn't able to make a code space" 
![Screenshot 2022-09-17 6 11 42 PM](https://user-images.githubusercontent.com/78823833/190879227-89caf1cc-2e9a-4ac4-8e88-33054ae172e2.png)

**Comment by samruddhikhandale** [comment]
Hi,
This is expected when the user does not have access to GitHub Codespaces for the particular repository. 

You may want to refer to [creating-a-codespace#access-to-github-codespaces](https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace#access-to-github-codespaces) for more details.
